components/web/components/org.wso2.carbon.apimgt.publisher.commons.ui/src/main/fragments/api-resource-row/api-resource-row.hbs

```diff
@@ -0,0 +1,51 @@
+<div class="resource_body_padding add-margin-bottom-2x">
+    <div class="add-margin-bottom-4x">
+        <strong>{{i18n "Description"}} : </strong><a class="notes" data-path="{{@params.resource_path}}" data-attr="description" title="description">{{#if @params.description}}{{ @params.description }}{{/if}}</a>
+    </div>
+    <table class="table-fullwidth add-margin-bottom-4x">
+        <tr>
+            <td width="40%"><strong>{{i18n "Produces"}} : </strong><a href="#" data-path="{{@params.resource_path}}" data-attr="produces" data-attr-type="comma_seperated" class="content_type produces" data-type="text" data-pk="1" data-title="Produces" title="Produces">{{#if @params.produces}}{{ @params.produces }}{{/if}}</a></td>
+            <td><strong>{{i18n "Consumes"}} : </strong><a href="#" data-path="{{@params.resource_path}}" data-attr="consumes" data-attr-type="comma_seperated" class="content_type consumes" data-type="text" data-pk="1" data-title="Consumes" title="Consumes">{{#if @params.consumes}}{{ @params.consumes }}{{/if}}</a></td>
+        </tr>
+    </table>
+    <div class="add-margin-bottom-4x">
+        <div class="add-margin-bottom-2x">
+            <strong>{{i18n "Parameters"}} :</strong>
+        </div>
+        {{#if @params.parameters}}
+        <table class="table table-condensed table-bordered">
+            <tr>
+                <th width="15%">{{i18n "Parameter Name"}}</th>
```

Comment:
i18n Keys should be in the following format with dots as separators.
Parameter.Name
Parameter.Type